src/ImageDeleteModal.jsx

```diff
@@ -17,7 +17,7 @@ const ImageDeleteModal = (props) => {
                 </Modal.Body>
                 <Modal.Footer>
                     <Button onClick={props.handleCancelImageDeleteModal}>Cancel</Button>
-                    <Button bsStyle="danger" onClick={props.handleRemoveImage}>{_("Delete")}</Button>{' '}
+                    <Button className="btn-img-delete" bsStyle="danger" onClick={props.handleRemoveImage}>{_("Delete")}</Button>{' '}
```

Comment:
Use id for navigatingin UI from the tests. classNames are supposed to use for css for multiple elements.